Marlin/Configuration_adv.h

```diff
@@ -845,6 +845,7 @@
  */
 //#define FWRETRACT  // ONLY PARTIALLY TESTED
 #if ENABLED(FWRETRACT)
+  //#define FW_SWAPPING                   // Allow swapping 'For separated extruders or mixing , but not for singlenozzle with toolchange process'
```

Comment:
The word here should be `separate`, not `separated`. And the description is confusing.